ArucoDetectRos-Amir's_Changes.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python3
+from __future__ import division  #in order to be able to set the (1/30)
+"""
+Framework   : OpenCV Aruco
+Description : Calibration of camera and using that for finding pose of multiple markers
+Status      : Working
+References  :
+    1) https://docs.opencv.org/3.4.0/d5/dae/tutorial_aruco_detection.html
+    2) https://docs.opencv.org/3.4.3/dc/dbb/tutorial_py_calibration.html
+    3) https://docs.opencv.org/3.1.0/d5/dae/tutorial_aruco_detection.html
+"""
+
+from math import floor
+import numpy as np
```

Comment:
What should be the standard when importing numpy?

1. Should we do `import numpy as np`
2. Or should we leave it as `import numpy`?